### PR TITLE
Week4 av nav drawer

### PR DESCRIPTION
**Pull request by avalencia0512**

### CODE REVIEW

**Comment by avalencia0512** [comment]
Pulled the most recent version of main. Resolved merge conflicts. Did not add nav drawer to StepCounterActivity.java because implementing NavigationView.OnNavigationItemSelectedListener caused errors in the code

File: app/src/main/java/com/personalchef/mealplan/MainActivity.java [hunk 1]
```diff
@@ -105,8 +127,48 @@ private void showSummaryNotification() {
...
+        }
+        startActivity(intent);
+
+
+        //Close drawer when user selects option
+        drawer = (DrawerLayout) findViewById(R.id.my_drawer_layout);
+        if (drawer.isDrawerOpen(GravityCompat.START)) {
+            drawer.closeDrawer(GravityCompat.START);
+        } else {
+            super.onBackPressed();
```
Comment (tdalia) [inline app/src/main/java/com/personalchef/mealplan/MainActivity.java]: What does this else part do?

File: app/src/main/java/com/personalchef/mealplan/LaunchActivity.java [hunk 2]
```diff
@@ -7,6 +7,7 @@
 import android.os.Bundle;
 import android.provider.Settings;
 import android.util.Log;
+import android.widget.TextView;
```
Comment (tdalia) [inline app/src/main/java/com/personalchef/mealplan/LaunchActivity.java]: Remove changes from this file.

File: .idea/misc.xml [hunk 3]
```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="ProjectPlainTextFileTypeManager">
+    <file url="file://$PROJECT_DIR$/app/src/main/res/layout/fragment_first.xml" />
```
Comment (tdalia) [inline .idea/misc.xml]: File doesn't exsits. Clean the code.

**Review by Agingalus — APPROVED** [review]

**Pull request merged**